Makefile

```diff
@@ -65,6 +65,7 @@ DIST_DIR := $(shell mktemp -d /tmp/glew.XXXXXX)/$(DIST_NAME)
 #   - use LN= on gmake command-line
 
 AR      ?= ar
+ARFLAGS.STATIC ?= cr
```

Comment:
Yes, agreed.  It does make sense to default to these.
I do still think it should be named ARFLAGS, since these flags are always for static library targets, not shared libraries.